### PR TITLE
Hook 'new' into the CLI telemetry pipeline

### PR DESCRIPTION
**Pull request by mlorbetske**
Hooks up the telemetry pipeline from the CLI for `dotnet new`

### CODE REVIEW

File: src/dotnet/commands/dotnet-new/NewCommandShim.cs [hunk 1]
```diff
@@ -24,7 +27,16 @@ internal class NewCommandShim
 
         public static int Run(string[] args)
         {
-            return New3Command.Run(CommandName, CreateHost(), new TelemetryLogger(null), FirstRun, args);
+            var sessionId = Environment.GetEnvironmentVariable(MSBuildForwardingApp.TelemetrySessionIdEnvironmentVariableName);
+            var telemetry = new Telemetry(new NuGetCacheSentinel(new CliFallbackFolderPathCalculator()), sessionId);
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-new/NewCommandShim.cs]: We have a MSBuildLogger that is very similar to this. Have you looked into it to see if you could simply re-use that?

Comment (mlorbetske) [inline src/dotnet/commands/dotnet-new/NewCommandShim.cs]: @livarcocc this was copied from the MSBuild one, I'll take a look to see if we can reuse that one all up

Comment (mlorbetske) [inline src/dotnet/commands/dotnet-new/NewCommandShim.cs]: @livarcocc the MSBuildLogger looks like it adapts to the MSBuild type of logging, these two lines are all that are in common between the two - should we make a helper to share between the two?

Comment (livarcocc) [inline src/dotnet/commands/dotnet-new/NewCommandShim.cs]: Should be all right for now. LGTM.

**Pull request merged**